### PR TITLE
Tk tutor csv upload

### PR DESCRIPTION
**Pull request by tkomarlu**

### CODE REVIEW

File: src/main/java/edu/ucsb/cs56/ucsb_courses_search/entities/CSVToTutors.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    .build();
+
+            Iterator<TutorBean> csvTutorIterator = csvToBean.iterator();
+            List<Tutor> tutors = new ArrayList<>();
+            while(csvTutorIterator.hasNext()) {
+                TutorBean tutorBean = csvTutorIterator.next();
+                Tutor tutor = new Tutor(tutorBean);
+                tutors.add(tutor);
+                //logger.info(tutor.toString());
+                //logger.info(tutorBean.getId() + " " + tutorBean.getFname() + " " + tutorBean.getLname() + " " + tutorBean.getEmail());
```
Comment (yinonrousso) [inline src/main/java/edu/ucsb/cs56/ucsb_courses_search/entities/CSVToTutors.java]: Code left as comments are generally "code smells". Its best to remove before merging to master.

**Review by yinonrousso — CHANGES_REQUESTED** [review]
LGTM just remove commented out code

**Review by yinonrousso — APPROVED** [review]
Looks good

File: src/main/config/localhost/data.sql [hunk 2]
```diff
@@ -6,5 +6,4 @@ INSERT INTO tutor (fname,lname,email) VALUES ('Scott','Chow','scottpchow@example
 INSERT INTO tutor (fname,lname,email) VALUES ('Zach','Sisco','zachsisco@example.org');
 INSERT INTO tutor (fname,lname,email) VALUES ('Yinon','Rousso','yinonRousso@example.org');
 INSERT INTO tutor (fname,lname,email) VALUES ('Kate','Perkins','kateperkins@example.org');
-INSERT INTO tutor (fname,lname,email) VALUES ('George','Kripac','Georgekripac@example.org');
-INSERT INTO tutor_assignment (course_offering_id,tutor_id) VALUES (1,1);
\ No newline at end of file
+INSERT INTO tutor (fname,lname,email) VALUES ('George','Kripac','Georgekripac@example.org');
```
Comment (scottpchow23) [inline src/main/config/localhost/data.sql]: Why did we delete the tutor_assignment?

Comment (tkomarlu) [reply src/main/config/localhost/data.sql]: I had removed all of the SQL inserts to test the upload button. I forgot to add this back.

**Pull request merged**